### PR TITLE
Add GameType filter, and add ForgeAPI Verify Files to validate Mod Downloads

### PR DESCRIPTION
**Pull request by shotah**

### CODE REVIEW

File: scripts/start-setupForgeApiMods [hunk 1]
```diff
@@ -192,11 +194,11 @@ if [ "$MODS_FORGEAPI_FILE" ] && [ -z "$MODS_FORGEAPI_PROJECTIDS" ]; then
   fi
 
   # Needs loop here to look up release types befor calling download.
-  jq -n "$required_dependencies" | jq -c '.[]?' | while read current_project; do
+  jq -c '.[]?' $MODS_FORGEAPI_FILE | while read current_project; do
```
Comment (shotah) [inline scripts/start-setupForgeApiMods:197]: Bug that was fixed.

File: scripts/start-setupForgeApiMods [hunk 2]
```diff
@@ -97,6 +97,7 @@ modFileByProjectID(){
     # Checking for a individual release type input, if not use global
     if [ $project_id_release_type ]; then
       updateReleaseNumber $project_id_release_type
+      unset $project_id_release_type
```
Comment (shotah) [inline scripts/start-setupForgeApiMods:100]: Bug that was fixed.

**Comment by shotah** [comment]
@itzg here's the PR.

File: scripts/start-setupForgeApiMods (hunk 2 above)
Comment (itzg) [reply scripts/start-setupForgeApiMods:100]: It looks like `unset` takes a variable name, so would be
```suggestion
      unset project_id_release_type 
```

**Review by itzg — COMMENTED** [review]
Just the `unset` change I think is needed

**Comment by itzg** [comment]
> I wonder if there is a way to be more verbose in the docker file

Specifically do you mean the docker compose file in each test directory?

File: scripts/start-setupWorld [hunk 3]
```diff
@@ -34,6 +34,7 @@ if [[ "$WORLD" ]] && ( isTrue "${FORCE_WORLD_COPY}" || [ ! -d "$worldDest" ] );
     # Stage contents so that the correct subdirectory can be picked off
     mkdir -p /tmp/world-data
     if ! extract "$WORLD" /tmp/world-data; then
+      log "ERROR extracting world from $WORLD"
```
Comment (itzg) [inline scripts/start-setupWorld:37]: Thanks!

**Review by itzg — APPROVED** [review]
Awesome!

**Pull request merged**